content/docs/10-audit-logs/05-kube-api-audit-logging.md

```diff
@@ -0,0 +1,209 @@
+The audit policy you create will capture all requests at the *metadata* level. To learn more about the various audit levels, visit the Kubernetes API [Audit Policy](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy) documentation.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Identify one of your cluster contro-plane nodes. You find a cluster node by navigating to the left **Main Menu** and selecting **Clusters**. Click on your cluster to access the details pages and click on the **Nodes** tab. The tab contains information about each pool, select a node from the **Master Pool** to view its IP address.
+
+
+3. SSH into one of your control-plane nodes by using its IP address and the SSH key you specified during the cluster creation process. 
+
+
+
+4. From a control-plane node in the target cluster, issue the following command to create your audit policy file.
+
+  <br />
+
+  ```bash
+  cat << EOF > /etc/kubernetes/audit-policy.yaml
+    apiVersion: audit.k8s.io/v1
+    kind: Policy
+    rules:
+    - level: Metadata
+  EOF
+  ```
+  The audit log output will be written to a file located at **/var/log/kubernetes/audit/audit.log**. In production environments, you should ensure this log file is ingested by a logging and monitoring application. 
+
+  The **/var/log/kubernetes/audit/** directory should be backed by persistent storage to ensure that any unshipped audit logs will not be lost during an unexpected outage of the node.
+
+<br />
+
+5. Next, you will update the Kubernetes API server manifest file. The manifest file is located in the **/etc/kubernetes/manifests** folder. 
+Before modifying the manifest file, create a backup copy. 
+
+  <br />
+
+  ```shell
+  cp /etc/kubernetes/manifests/kube-apiserver.yaml  /etc/kubernetes/manifests/kube-apiserver.backup 
+  ```
+
+6. Now that you have a backup copy of the manifest file go ahead and open up the file **/etc/kubernetes/manifests/kube-apiserver.yaml** in a text editor such as Vi or Nano.
+
+  <br />
+
+  ```shell
+  vi /etc/kubernetes/manifests/kube-apiserver.yaml
+  ```
+
+  Append the following YAML configuration to your kube-apiserver manifest.
+
+  <br />
+
+  ```yaml
+    volumeMounts:
+      - mountPath: /etc/kubernetes/audit-policy.yaml
+        name: audit
+        readOnly: true
+      - mountPath: /var/log/kubernetes/audit/
+        name: audit-log
+        readOnly: false
+    volumes:
+    - name: audit
+      hostPath:
+        path: /etc/kubernetes/audit-policy.yaml
+        type: File
+    - name: audit-log
+      hostPath:
+        path: /var/log/kubernetes/audit/
+        type: DirectoryOrCreate
+  ```
+
+7. The next step is to update the Kubernetes API parameters with audit settings. 
+The Kubernetes API parameters are located toward the top of the file. The code snippet below showcases where in the file to place the parameters.
```

Comment:
The top of the file contains the Kubernetes API parameters. Refer to the code snippet below to determine where to place these parameters.